operator/builtin/input/udp/udp.go

```diff
@@ -53,6 +69,12 @@ func (c UDPInputConfig) Build(context operator.BuildContext) ([]operator.Operato
 		return nil, err
 	}
 
+	// If MaxLogSize not set, set sane default in order to remain
+	// backwards compatible with existing plugins and configurations
+	if c.MaxLogSize == 0 {
+		c.MaxLogSize = DefaultMaxLogSize
+	}
+
```

Comment:
`tcp_input` does a check against `minBufferSize`. We should probably do the same here. Also, since we renamed `MaxBufferSize` to `MaxLogSize`, we should probably call this `minLogSize`.

```go
if c.MaxLogSize < minBufferSize {
	return nil, fmt.Errorf("invalid value for parameter 'max_log_size', must be equal to or greater than %d bytes", minBufferSize)
}
```